FIAT/hdiv_trace.py

```diff
@@ -16,14 +16,19 @@
 # along with FIAT. If not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import absolute_import, print_function, division
+from six import iteritems, itervalues
+from six.moves import range
 
 import numpy as np
-from FIAT.discontinuous_lagrange import DiscontinuousLagrange
-from FIAT.reference_element import ufc_simplex
+from FIAT.reference_element import (ufc_simplex, POINT,
+                                    LINE, QUADRILATERAL,
+                                    TRIANGLE, TETRAHEDRON,
+                                    TENSORPRODUCT)
 from FIAT.functional import PointEvaluation
 from FIAT.polynomial_set import mis
-from FIAT import FiniteElement
-from FIAT import dual_set
+from FIAT import (FiniteElement, dual_set,
+                  DiscontinuousLagrange,
+                  TensorProductElement)
```

Comment:
That's better. For completeness, one should check that `FIAT.finite_element` and `FIAT.hdiv_trace` don't mutually reference each other. When you `from FIAT import FiniteElement`, then you depend on `FIAT.__init__`. If in `__init__.py` you `from FIAT.hdiv_trace import ...` then you have a circular dependency, which may just work with the right ordering of imports, but I think `from FIAT.finite_element import FiniteElement` is more reliable.